<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-5EVK54D" failures="2" tests="8" name="Test" time="0.215" errors="0" timestamp="2022-12-21T08:18:22 IST">
  <testcase classname="testngdiscussion.FailedTestCaseExecution" name="login" time="0.012"/>
  <testcase classname="testngdiscussion.FailedTestCaseExecution" name="home" time="0.007">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testngdiscussion.FailedTestCaseExecution.home(FailedTestCaseExecution.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- home -->
  <testcase classname="testngdiscussion.FailedTestCaseExecution" name="timeline" time="0.0">
    <skipped/>
  </testcase> <!-- timeline -->
  <testcase classname="testngdiscussion.FailedTestCaseExecution" name="profile" time="0.004"/>
  <testcase classname="testngdiscussion.FailedTestCaseExecution" name="logout" time="0.008"/>
  <testcase classname="testngdiscussion.FailedTestCaseExecution2" name="editProfile" time="0.005"/>
  <testcase classname="testngdiscussion.FailedTestCaseExecution2" name="deleteProfile" time="0.005">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testngdiscussion.FailedTestCaseExecution2.deleteProfile(FailedTestCaseExecution2.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- deleteProfile -->
  <testcase classname="testngdiscussion.FailedTestCaseExecution2" name="closeBrowser" time="0.006"/>
</testsuite> <!-- Test -->
